app/client/src/components/shared/MapWidgets.tsx

```diff
@@ -1714,255 +1795,567 @@ function ShowSurroundingMonitoringLocations({
+  setLastHuc12: Dispatch<SetStateAction<string>>,
+  setUpstreamExtent: Dispatch<SetStateAction<__esri.Viewpoint>>,
+  setUpstreamLayer: Dispatch<SetStateAction<__esri.Layer>>,
+  setUpstreamLayerVisible: Dispatch<SetStateAction<boolean>>,
+  setUpstreamWidgetDisabled: Dispatch<SetStateAction<boolean>>,
+  view: __esri.MapView | null,
+  setUpstreamLoading: Dispatch<SetStateAction<boolean>>,
+  huc12 = null,
+  canDisable = true,
+  setError?: (isError: boolean) => void,
+) {
+  // if widget is disabled do nothing
+  if (getUpstreamWidgetDisabled()) return;
+  const upstreamLayer = getUpstreamLayer();
+  if (!upstreamLayer) return;
+
+  const currentHuc12 = huc12 ?? getHuc12();
+  // if location changed since last widget click, update lastHuc12 state
+  if (currentHuc12 !== lastHuc12) {
+    setLastHuc12(currentHuc12);
+    setErrorMessage('');
+    upstreamLayer.error = false;
+  }
+
+  // already encountered an error for this location - don't retry
+  if (upstreamLayer.error === true) {
+    setError?.(true);
+    return;
+  }
+
+  // if upstream layer is displayed, zoom to
+  // current location extent and hide the upstream layer
+  if (
+    currentHuc12 === lastHuc12 &&
+    upstreamLayer.visible &&
+    upstreamLayer.graphics.length > 0
+  ) {
+    const currentExtent = getCurrentExtent();
+    currentExtent && view?.goTo(currentExtent);
+    view?.popup.close();
+    upstreamLayer.visible = false;
+    setUpstreamLayerVisible(false);
+    setUpstreamLayer(upstreamLayer);
+    return;
+  }
+
+  // if upstream layer is hidden, zoom to full
+  // upstream extent and display the upstream layer
+  if (
+    currentHuc12 === lastHuc12 &&
+    !upstreamLayer.visible &&
+    upstreamLayer.graphics.length > 0
+  ) {
+    view?.goTo(getUpstreamExtent());
+    upstreamLayer.visible = true;
+    setUpstreamLayerVisible(true);
+    setUpstreamLayer(upstreamLayer);
+    return;
+  }
+
+  // fetch the upstream catchment
+  const filter = `xwalk_huc12='${currentHuc12}'`;
+
+  setUpstreamLoading(true);
+
+  if (services.status !== 'success') return;
+  const url = services.data.upstreamWatershed;
+
+  const queryParams = {
+    returnGeometry: true,
+    where: filter,
+    outFields: ['*'],
+    signal: abortSignal,
+  };
+  query
+    .executeQueryJSON(url, queryParams)
+    .then((res) => {
+      setUpstreamLoading(false);
+      const watershed = getWatershed() || 'Unknown Watershed';
+      const upstreamTitle = `Upstream Watershed for Currently Selected Location: ${watershed} (${currentHuc12})`;
+
+      if (!res || !res.features || res.features.length === 0) {
+        upstreamLayer.error = true;
+        upstreamLayer.graphics.removeAll();
+        setUpstreamLayer(upstreamLayer);
+        canDisable && setUpstreamWidgetDisabled(true);
+        setUpstreamLayerVisible(false);
+        setErrorMessage(
+          `Unable to get upstream watershed data for ${
+            huc12 ? 'the selected' : 'this'
+          } location.`,
+        );
```

Comment:
```suggestion
        setErrorMessage(
          `No upstream watershed data available for ${
            huc12 ? 'the selected' : 'this'
          } location.`,
        );
```

I think this should be a `No data available` message and be an info level message. This language sounds like there was a problem, when really there just isn't any data available. We should do the same thing for the community page as well.

I think it is expected for some locations to not have any upstream watersheds. Especially for watersheds in the desert. 


To reproduce go to http://localhost:3000/tribe/PUEBLO_SANTAANA and then follow the GIF below.
![no-upstream-watershed](https://user-images.githubusercontent.com/46329268/198295850-ae5fbf59-b73f-40d2-adea-26b7c5cbc5f6.gif)